rclcpp_action/test/test_client.cpp

```diff
@@ -565,7 +565,7 @@ TEST_F(TestClientAgainstServer, async_send_goal_with_feedback_callback_wait_for_
 
   ASSERT_EQ(5u, wrapped_result.result->sequence.size());
   EXPECT_EQ(3, wrapped_result.result->sequence.back());
-  EXPECT_EQ(5, feedback_count);
+  EXPECT_EQ(2, feedback_count);
```

Comment:
Is this expected to be always `2` without any racy condition?

according to https://github.com/ros2/rclcpp/blob/f12e3c69dcc5823b474d43ae1d7bf3c596154cbd/rclcpp_action/test/test_client.cpp#L109-L114, it could be more than `2` if feedback data is only available before other entities?

```suggestion
  EXPECT_GE(2, feedback_count);
```

Reply:
True, it can be greater (or equal) than 2. Fixed in last commit